packages/moonstone/VideoPlayer/MediaSliderDecorator.js

```diff
@@ -125,11 +126,10 @@ const MediaSliderDecorator = hoc((config, Wrapped) => {	// eslint-disable-line n
 
 		move (clientX) {
 			this.setState((state) => {
-				if (clientX >= state.minX && clientX <= state.maxX) {
-					return {
-						x: calcProportion(state.minX, state.maxX, clientX)
-					};
-				}
+				const value = clamp(state.minX, state.maxX, clientX);
+				return {
```

Comment:
I think we _could_ optimize this to return `null` when `x` isn't changing but I doubt it'll make a material difference given that the slider will already be updating frequently during playback. Noting for future consideration only.